gitpod-io/gitpod
# [server] Guard against bogus RPC method names

## Description


## Related Issue(s)

Fixes #

## How to test
 - singup to the preview env
 - start a workspace, and see that everything works as usual

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:
